src/index.html

```diff
@@ -7,11 +7,178 @@
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>HTML Form</title>
+    <title>Personal information</title>
     <link rel="stylesheet" href="./style.css">
+    <script type="text/javascript" src="./main.js"></script>
+
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+
+    <form>
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <label for="surname" class="labelWithMargin">
+          Surname:
+          <input
+            type="text"
+            name="usersurname"
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label for="name" class="labelWithMargin">
+          Name:
+          <input
+            type="text"
+            name="username"
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label for="old" class="labelWithMargin">
+           How old are You?
+          <input
+            type="number"
+            name="userage"
+            min="1"
+            max="100"
+            value="12"
+            autocomplete="off"
+          >
+        </label>
+
+
+        <label for="old" class="labelWithMargin">
+          Full date of birth:
+          <input
+            type="date"
+            name="userbirth"
+          >
+        </label>
+
+
+        <label for="old" class="labelWithMargin">
+           I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="iacc"
+          >
+        </label>
+
+      </fieldset>
+
+
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label for="surname" class="labelWithMargin">
+          E-mail:
+          <input type="email"
+            name="email"
+            autocomplete="off"
+            placeholder="email@example.com"
+          >
+        </label>
+
+
+        <label for="password" class="labelWithMargin">
+          Password:
+          <input type="password"
+            name="password"
+            autocomplete="off"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <label for="cats" class="labelWithMargin">
+            Do you love cats?
+          <label>
+            <input type="radio"
+              name="cat"
+              value="yes"
+            > Yes
+          </label>
+
+          <label >
+            <input type="radio"
+              name="cat"
+              value="no"
+            > No
+          </label>
+        </label>
+
+        <label for="color" class="labelWithMargin">
+          What is your favorit color?
+          <input type="color"
+           name="impcolor"
+          >
```

Comment:
```suggestion
          <input
            type="color"
            name="impcolor"
          >
```
check your code style with checklist